art/estimators/regression/pytorch.py

```diff
@@ -533,6 +533,45 @@ def compute_loss(  # type: ignore # pylint: disable=W0221
 
         return loss.detach().cpu().numpy()
 
+    def compute_loss_from_predictions(  # type: ignore # pylint: disable=W0221
```

Comment:
Why do we need this new method in addition to the existing method `compute_losses`?